packages/office-ui-fabric-react/src/components/Button/SplitButton/SplitButton.styles.ts

```diff
@@ -40,15 +47,14 @@ export const getStyles = memoizeFunction(
 
               selectors: {
                 [HighContrastSelector]: {
-                  color: 'Window',
-                  backgroundColor: 'WindowText',
+                  color: 'WindowText',
+                  backgroundColor: 'Window',
+                  border: '1px solid WindowText',
+                  borderRightWidth: '0',
                   MsHighContrastAdjust: 'none',
                 },
               },
             },
-            '.ms-Button--primary + .ms-Button': {
```

Comment:
you cannot remove this block, it's breaking non-HC: https://screener.io/v2/states/Skgr32fUL.microsoft-fluentui/a11y-commandbar

Reply:
whoops!